src/API.jl

```diff
@@ -34,7 +34,8 @@ develop(pkg::Union{AbstractString, PackageSpec}; kwargs...) = develop([pkg]; kwa
 develop(pkgs::Vector{<:AbstractString}; kwargs...) =
     develop([check_package_name(pkg, :develop) for pkg in pkgs]; kwargs...)
 develop(pkgs::Vector{PackageSpec}; kwargs...)      = develop(Context(), pkgs; kwargs...)
-function develop(ctx::Context, pkgs::Vector{PackageSpec}; shared::Bool=true, kwargs...)
+function develop(ctx::Context, pkgs::Vector{PackageSpec};
+                 shared::Bool=true, keep_manifest::Bool=false, kwargs...)
```

Comment:
What does `keep_manifest` do? Which manifest is kept?

Reply:
The manifest created here: https://github.com/JuliaLang/Pkg.jl/pull/1033/files#diff-920ed338dcbd9d62489c6d12dc2fbb18R1172. Together, its pretty much these two lines which support #1030